packages/clerk-js/src/ui/localization/defaultEnglishResource.ts

```diff
@@ -261,13 +280,35 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
         resendButton: 'Resend link',
         successMessage: 'The email {{identifier}} has been added to your account.',
       },
+      removeResource: {
+        title: 'Remove email address',
+        successMessage: '{{emailAddress}} has been removed from your account.',
+      },
+    },
+    phoneNumberPage: {
+      title: 'Add phone number',
+      successMessage: '{{identifier}} has been added to your account.',
+      removeResource: {
```

Comment:
I think that most components in RemoveResourcePage these strings as props: 
```
  title: string;
  messageLine1: string;
  messageLine2: string;
  successMessage: string;
```
Can we add the missing `messageLine1` and `messageLine2` keys?

Note: this applies to all other resources as well.